server/routes/adjudications/index.test.ts

```diff
@@ -65,7 +67,7 @@ describe('GET /adjudications', () => {
     const res = await request(app).get('/adjudications')
 
     expect(res.status).toBe(200)
-    expect(mockServices.adjudicationsService.getReportedAdjudicationsFor).toHaveBeenCalledWith('12345', '12345', 'en')
+    expect(mockServices.adjudicationsService.getReportedAdjudicationsFor).toHaveBeenCalledWith('12345', 'CKI', 'en')
```

Comment:
Need some checks for the new code to ensure it really does redirect based in prisoner number 👍